nixos/modules/programs/gnupg.nix

```diff
@@ -6,6 +6,20 @@ let
 
   cfg = config.programs.gnupg;
 
+  xserverCfg = config.services.xserver;
+
+  defaultPinentryFlavor =
+    if xserverCfg.desktopManager.lxqt.enable
+    || xserverCfg.desktopManager.plasma5.enable then
+      "qt"
+    else if xserverCfg.desktopManager.xfce.enable
+    || xserverCfg.desktopManager.xfce4-14.enable then
```

Comment:
Xfce 4.14 is the first one to support GTK 3, only `xfce` should be here. Although, maybe we should have renamed the option to `xfce4-12` to avoid regressions in the future.

Reply:
I do not know. Maybe we should just let Xfce 4.12 default to GTK 3 too, it’s not like it won’t use any GTK 2 programs.

Reply:
Well, I won't rename the option here :-D Removed the `xserverCfg.desktopManager.xfce4-14.enable` option though.

Reply:
@worldofpeace what do you think about using GTK3 with Xfce 4.12 too?

Reply:
No problem. I have to remove it anyway.